sdk/aqueduct/api_client.py

```diff
@@ -98,6 +99,7 @@ class APIClient:
     LIST_INTEGRATIONS_ROUTE = "/api/integrations"
     LIST_TABLES_ROUTE = "/api/tables"
     GET_WORKFLOW_ROUTE_TEMPLATE = "/api/workflow/%s"
+    GET_WORKFLOW_OBJECTS_ROUTE = "/api/workflow/%s/objects"
```

Comment:
`GET_WORKFLOW_SAVED_OBJECTS_ROUTE`?

Reply:
and we'd name the api client method consistently.

Reply:
Ok, sounds good. Would updating the endpoint to `/api/workflow/%s/saved_objects` make it clearer?

Reply:
Also reminder to refer to this doc for endpoint naming. Typically endpoints shouldn't consist of `_` and `objects` seems fine to me.

Reply:
Thanks for the helpful resource @cw75. I'll keep the endpoint as "/api/workflow/%s/objects" and modify `GET_WORKFLOW_OBJECTS_ROUTE` to `LIST_WORKFLOW_OBJECTS_ROUTE` to be consistent with the url & @likawind 's suggestion to rename the function to `list_saved_objects` as per the [naming guide](https://www.notion.so/aqueducthq/SDK-External-API-Naming-ce685c0980ae4d7ebee569bf53ef2a01).